tests/acceptance/features/apiSpaces/lockSpaces.feature

```diff
@@ -0,0 +1,92 @@
+@api @skipOnOcV10
+Feature: lock
+  # Note: This Feature includes all the tests from core (apiWebdavLock suite) related to /Shares since in core no implementation is there for space Shares Jail
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+      | Carol    |
+    And user "Alice" has created folder "PARENT"
+    And user "Brian" has created folder "PARENT"
+    And user "Carol" has created folder "PARENT"
+    And using spaces DAV path
+
+
+  Scenario Outline: lock should propagate correctly when uploaded to a reshare that was locked by the owner
+    Given user "Alice" has shared folder "PARENT" with user "Brian"
+    And user "Brian" has accepted share "/PARENT" offered by user "Alice"
+    And user "Brian" has shared folder "Shares/PARENT" with user "Carol"
```

Comment:
Maybe we can add a step for resharing through spaces as well?

Reply:
i will check if its possible. @SwikritiT